stable/openvpn/README.md

```diff
@@ -15,18 +15,20 @@ Wait for the external load balancer IP to become available.  Check service statu
  
 Please be aware that certificate generation is variable and may take some time (minutes).
 Check pod status via:
+```bash
 POD_NAME=`kubectl get pods -l type=openvpn | awk END'{ print $1 }'` \
 && kubectl log $POD_NAME --follow
+```
 
 When ready generate a client key as follows:
 
 ```bash
-POD_NAME=`kubectl get pods --namespace {{ .Release.Namespace }} -l type=openvpn | awk END'{ print $1 }'` \
-&& SERVICE_NAME=`kubectl get svc --namespace {{ .Release.Namespace }} -l type=openvpn | awk END'{ print $1 }'` \
-&& SERVICE_IP=`kubectl get svc $SERVICE_NAME --namespace {{ .Release.Namespace }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}'` \
+POD_NAME=`kubectl get pods --namespace default -l type=openvpn | awk END'{ print $1 }'` \
```

Comment:
I wasn't able to parse the brackets, the error complained about reading '{' as input. Might be just BSD specific. I'll revert it.